src/libaktualizr/bootloader/bootloader.h

```diff
@@ -29,6 +29,7 @@ class Bootloader {
 
   INvStorage& storage_;
   boost::filesystem::path reboot_sentinel_;
+  std::string reboot_command_;
```

Comment:
It is only used in one place, so it probably doesn't need to be an attribute, simply a variable in the concerned method if needed.